frontend/src/components/metadata/DisplayMetadata.tsx

```diff
@@ -13,6 +13,7 @@ import {
 import { Agent } from "./Agent";
 import { MetadataDeleteButton } from "./widgets/MetadataDeleteButton";
 import { fetchPublicDatasetMetadata } from "../../actions/public_dataset";
+import { Link as RouterLink } from "react-router-dom";
```

Comment:
I didn't find where the RouterLink is being used. Could you remove that?

Reply:
removed.